Source/WebCore/page/LocalDOMWindow.cpp

```diff
@@ -1633,7 +1633,9 @@ RefPtr<CSSRuleList> LocalDOMWindow::getMatchedCSSRules(Element* element, const S
     auto [pseudoElementIsParsable, pseudoElementIdentifier] = CSSSelectorParser::parsePseudoElement(pseudoElement, parserContext);
     if (!(pseudoElementIsParsable || (pseudoElementIdentifier && !pseudoElementIdentifier->nameArgument.isNull())) && !pseudoElement.isEmpty())
         return nullptr;
-    auto pseudoId = pseudoElementIdentifier ? pseudoElementIdentifier->pseudoId : PseudoId::None;
+    std::optional<PseudoId> pseudoId;
+    if (pseudoElementIdentifier)
+        pseudoId = pseudoElementIdentifier->pseudoId;
```

Comment:
Since `pseudoStyleRulesForElement` takes in a `std::optional<Style::PseudoElementIdentifier>`, I'd just pass `pseudoElementIdentifier` (which is already the right type) where we're currently passing the `pseudoId`

Reply:
We can then entirely get rid of the pseudoId variable.

Reply:
Whoops. You are right. Thank you. I will update the pull request.

Reply:
Updated.